Update deployment guides to use volumes (#1559)
n8n-io/n8n-docs: _snippets/self-hosting/installation/docker-compose-updating.md

> If you're running n8n using a Docker Compose file, follow the steps below to update n8n.

Reviewer: ```suggestion
If you run n8n using a Docker Compose file, follow these steps to update n8n:
```
